tutorials/migration.md

```diff
@@ -796,21 +679,18 @@ becomes
 Switch S_L_Family "Family Room Lamp" <light> {channel="zwave:device:528f7aca:node10:switch_binary"}
 ```
 
-Congratulations, you are now using the 2.0 version of the binding. Assuming the
-behaviors of the binding are the same, there should be no required changes to your
-Rules, Sitemaps, or Persitence. Test your Items each step of the way to verify
-they are working.
+Congratulations, you are now using the 2.x version of the binding. 
+Assuming the behaviors of the binding are the same, there should be no required changes to your Rules, Sitemaps, or Persitence. 
+Test your Items each step of the way to verify they are working.
 
 ### Manually Creating Things
 
-Not all 2.0 bindings support automatic discovery of Things or by their very nature
-require manual creation of Things. As with linking Channels to Items, there is
-more than one way to do this including through PaperUI, Habmin, the REST API, or
-through text files. This tutorial will only cover creating Things in text files as
-that will be closer to the openHAB 1.x way of doing things.
-
-All definitions of new Things are written to .things files in the
-`<openHAB 2 conf>/things` folder. The syntax for a Thing definition varies from
-binding to binding. See the binding's readme for the specific format and parameters
-required. Also see the binding's readme for the list of channels the Thing
-supports. As discussed above, the Channel ID will be the `<Thing ID>#<channel>`.
+Not all 2.0 bindings support automatic discovery of Things or by their very nature require manual creation of Things. 
+As with linking Channels to Items, there is more than one way to do this including through PaperUI, Habmin, the REST API, or through text files. 
+This tutorial will only cover creating Things in text files as that will be closer to the openHAB 1.x way of doing things.
+
+All definitions of new Things are written to .things files in the `<openHAB 2 conf>/things` folder. 
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.